src/transformers/models/gptj/configuration_gptj.py

```diff
@@ -135,3 +139,71 @@ def __init__(
         super().__init__(
             bos_token_id=bos_token_id, eos_token_id=eos_token_id, tie_word_embeddings=tie_word_embeddings, **kwargs
         )
+
+
+class GPTJOnnxConfig(OnnxConfigWithPast):
```

Comment:
Is this class copied from `GPT2OnnxConfig`? Is yes, you should add the following comment above the class:

```
# Copied from transformers.models.gpt2.configuration_gpt2.GPT2OnxxConfig
```

followed by running 

```
make fix-copies
```

This will ensure that any changes to the source class will be propagate to the copied ones :)